app/jobs/notify_subscriber.rb

```diff
@@ -35,7 +35,7 @@ def send_message_to_webhook(webhook_url, submission)
   end
 
   def delete_subscriber(subscriber)
-    deletion_threshold = ENV["SUBSCRIBER_FAILED_ATTEMPT_DELETION_THRESHOLD"]
+    deletion_threshold = ENV.fetch("SUBSCRIBER_FAILED_ATTEMPT_DELETION_THRESHOLD", nil)
```

Comment:
Can we include details of the purpose of these changes in the pull request? They don't seem to be related to the ticket.

Reply:
This change doesn't actually do anything - in ruby, `[x]` is the same thing as `.fetch(x, nil)`.  I'm guessing this David has some code formatter plugin in his IDE that's configured to prefer fetch?